homeworks/01-bubble-sort/src/components/Button/index.tsx

```diff
@@ -0,0 +1,14 @@
+import { FC, ComponentPropsWithoutRef } from "react";
+
+type Props = {
+  type?: "button" | "reset" | "submit";
```

Comment:
Redundant type because `ComponentPropsWithoutRef` includes it and anyway will replace this value. You can just remove it.